sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -323,9 +349,9 @@ class SQLContext(@transient val sparkContext: SparkContext)
 
     val sqlContext: SQLContext = self
 
-    def codegenEnabled = self.codegenEnabled
+    def codegenEnabled = self.conf.codegenEnabled
 
-    def numPartitions = self.numShufflePartitions
+    def numPartitions = self.conf.numShufflePartitions
```

Comment:
Maybe more future proof and slightly more clear? Since it is a small addition (just "self.").